### ISSUE TITLE
Data Question

### ISSUE THREAD

**Issue by haritsahm** [Nu-AI/Livid-About-COVID#4]
I'm confused with the data that you provided. What does active, total, pop, and med means and why are the value so small? And is the value next to the province/state name ex: shanghai 24240000 means its total population?

**Comment by craymichael** [comment]
Hi - thanks for the inquiry. 

The active and total case data is located in the respective CSV files in the data folder, but when they are used in the model they are normalized to much smaller values. Depending on the file you are looking at, they may be normalized differently but will be in a similar 0-to-1-ish range (e.g., by min-max normalization). 

Province/state name: Shanghai with 24240000 is the total population

Last to clarify what the names mean-
```
active: day-wise number of current active COVID-19 cases
total: day-wise number of cumulative total COVID-19 cases
pop: a constant population for an area (weighted population density in some files)
med: a constant number of hospitals per 100 people (if referring to 'HospBeds.csv' data)
```
Basically any small numbers you're seeing are due to normalization.

Does this answer your questions? Thanks!

**Comment by haritsahm** [comment]
Okay this is helpful! Another question, the fit_bexar is based on SIRNet and the notebooks are based on custom LSTM networks?

**Comment by craymichael** [comment]
Great! That is correct - fit_bexar uses SIRNet for learning and prediction while the notebooks use a preliminary model that is just a custom LSTM architecture (and different formatting of data, no mobility data).

**Comment by haritsahm** [comment]
Thank you so much!
